android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -109,7 +109,7 @@
 @ReactModule(name = RNCWebViewManager.REACT_CLASS)
 public class RNCWebViewManager extends SimpleViewManager<WebView> {
 
-  public static String activeUrl = null;
+  public static boolean isOverriddenLoad = false;
```

Comment:
Since you are fixing this code, it would be nice if you could also fix the abuse of the static keyword here. Object instances should not use class variables to store state. This pattern is a bug if there are ever 2 webview instances in the same app.

Reply:
Nice catch! Created a static class in `RNCWebView` instead, to inject the filter logic into `RNCWebViewClient` and `RNCWebChromeClient`.